meinberlin/apps/budgeting/assets/__tests__/ControlBar.jest.jsx

```diff
@@ -30,3 +30,18 @@ test('ControlBar shows filters', () => {
   const orderingFilterElement = screen.getByText('ordering: Most recent')
   expect(orderingFilterElement).toBeTruthy()
 })
+
+test('ControlBar expanded bar', async () => {
+  const onChangeFiltersFn = jest.fn()
+  render(
+    <ControlBar
+      filters={filters}
+      onChangeFilters={() => onChangeFiltersFn()}
+      numOfResults={2}
+    />
+  )
+  const filterButton = screen.getByLabelText('Toggle filters')
+  fireEvent.click(filterButton)
+  const expandedFilter = screen.getByText(/category1/)
```

Comment:
sorry, didn't think about that earlier but should we maybe check that it's not expanded before the click ? e.g. `expected(expandedFilter).toBeFalsy` ?